src/base/build/git.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+VERSION_CODENAME=$(. /etc/os-release && echo ${VERSION_CODENAME})
+
+echo "deb http://ppa.launchpad.net/git-core/ppa/ubuntu ${VERSION_CODENAME} main" | tee -a /etc/apt/sources.list.d/git.list
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E1DD270288B4E6030699E45FA1715D88E1DF1F24
+
+# TODO: Only latest version available on launpad :-/
```

Comment:
```suggestion
# TODO: Only latest version available on launchpad :-/
```

Is there any benefit to `GIT_VERSION` then? Just to trigger a new build?

Reply:
Initially i though i can use this to install a specific git version. Maybe in future. 🙃